sechub-doc/src/test/java/com/mercedesbenz/sechub/restdoc/SignupAdministrationRestControllerRestDocTest.java

```diff
@@ -123,7 +129,8 @@ public void restdoc_delete_signup() throws Exception {
 
         /* execute + test @formatter:off */
         this.mockMvc.perform(
-        		delete(apiEndpoint,"userId1")
+        		delete(apiEndpoint,"userId1").
+        			header(AuthenticationHelper.HEADER_NAME, AuthenticationHelper.getHeaderValue())
```

Comment:
Inside this method, the `requestHeaders(...)` part seems to be missing, because it does not occur inside the generated `openapi3.json` file.